ProjectFiles/Packages/System.Threading.Tasks.Extensions.4.5.1/LICENSE.TXT

```diff
@@ -0,0 +1,23 @@
+The MIT License (MIT)

```

Comment:
Was this supposed to be in the gitignore?

Reply:
I have honestly no idea. But that library is needed for tests I think so I will remove from this branch.